tools/src/publish-packages/tasks/packPackageToTarball.ts

```diff
@@ -31,7 +31,7 @@ export const packPackageToTarball = new Task<TaskArgs>(
           return;
         } catch (error) {
           step.fail();
-          logger.error(error.stderr);
+          logger.error(error);
```

Comment:
Was this change intentional? I think we still only want to show the actual error message, right? Or the whole object?

Reply:
Yes, intentional. It was showing `undefined` before since stderr didn't always exist I think, or at least for this bug. Changing this to print the whole error shows the full output.